burn-core/src/data/dataloader/builder.rs

```diff
@@ -26,21 +36,57 @@ where
         }
     }
 
+    /// Sets the batch strategy.
```

Comment:
`Sets the batch size to a fix number. The [fix batch strategy](FixBatchStrategy) will be used.`